.github/workflows/run.e2e-tests.yml

```diff
@@ -184,6 +184,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       IZE_EXAMPLES_PATH: ${{ github.workspace }}/examples/sls-apps-monorepo
+      ACTIONS_RUNNER_DEBUG: '${{ secrets.ACTIONS_RUNNER_DEBUG }}'
```

Comment:
I found that `RUNNER_DEBUG` is always set to `1` when GHA debug is running (per https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging).

Let's remove the env var config here (as it allways will be passed to the test process if we are running in DEBUG)